examples/tutorials/curation/plot_1_automated_curation.py

```diff
@@ -0,0 +1,248 @@
+#
+# Go to that page, and take a look at the ``Files``. The models are contained in the
+# `skops files <https://skops.readthedocs.io/en/stable/>`_ and there are *two* in this repo.
+# We can choose which to load in the ``load_model`` function as follows:
+#
+# .. code-block::
+#    import spikeinterface.curation as sc
+#    model, model_info = sc.load_model(
+#        repo_id = "AnoushkaJain3/curation_machine_learning_models",
+#        model_name= 'noise_neuron_model.skops',
+#    )
+#
+# If you run this locally you will receive an error:
+#
+# .. code-block::
+#
+#     UntrustedTypesFoundException: Untrusted types found in the file: ['sklearn.metrics._classification.balanced_accuracy_score', 'sklearn.metrics._scorer._Scorer', 'sklearn.model_selection._search_successive_halving.HalvingGridSearchCV', 'sklearn.model_selection._split.StratifiedKFold']
+#
+# This is a security warning. Sharing models, with are Python objects, is complicated.
+# We have chosen to use the `skops format <https://skops.readthedocs.io/en/stable/>`_, instead
+# of the common but insecure ``.pkl`` format (read about ``pickle`` security issues
+# `here <https://lwn.net/Articles/964392/>`_). While unpacking the ``.skops`` file, each function
+# is checked. Ideally, skops should recognise all `sklearn`, `numpy` and `scipy` functions and
+# allow the object to be loaded if it only contains these (and no unkown malicious code). But
+# when ``skops`` it's not sure, it raises an error. Here, it doesn't recognise
+# ``['sklearn.metrics._classification.balanced_accuracy_score', 'sklearn.metrics._scorer._Scorer',
+# 'sklearn.model_selection._search_successive_halving.HalvingGridSearchCV',
+# 'sklearn.model_selection._split.StratifiedKFold']``. Taking a look, these are all functions
+# from `sklearn`, and we can happily add them to the ``trusted`` functions to load:
+#
+# .. code-block::
+#
+#     model, model_info = sc.load_model(
+#         model_name = 'noise_neuron_model.skops',
+#         repo_id = "AnoushkaJain3/curation_machine_learning_models",
+#         trusted = ['sklearn.metrics._classification.balanced_accuracy_score', 'sklearn.metrics._scorer._Scorer', 'sklearn.model_selection._search_successive_halving.HalvingGridSearchCV', 'sklearn.model_selection._split.StratifiedKFold']
+#     )
+#
+# As ``skops`` continues to be developed, we hope more of these functions will be :code:`trusted`
+# by default.
+#
+# If you unequivocally trust the model (e.g. if you have created it), you can bypass this security
+# step by passing ``trust_model = True`` to the ``load_model`` function.
+#
+# In general, you should be cautious when downloading ``.skops`` files and ``.pkl`` files from repos,
+# especially from unknown sources.
+#
+#
+# Directly applying a sklearn Pipeline
+# ------------------------------------
+#
+# Instead of using ``HuggingFace`` and ``skops``, you might have another way of receiving a sklearn
```

Comment:
I was a bit confused by this, is 'might have another way' being a somone directly sends to you, or you download from github?

In th below code example, `model` is not defined. Can you remind here how to load `model` from file?